src/test/resources/soop_sst_trajectory.xml

```diff
@@ -6,68 +6,849 @@
+    </attribute>
+    <attribute name="comment" sql="select comment from $instance"/>   --????
+    <attribute name="references" value="'http://www.imos.org.au'"/>
+    <attribute name="vessel_name" value="select vessel_name from $instance"/>
+    <attribute name="platform_code" sql="select platform_code from $instance"/>
+    <attribute name="deployment_code" sql="select voyage_number from $instance"/>
+    <attribute name="featureType" value="'trajectory'"/>
+    <attribute name="naming_authority" value="'IMOS'"/>
+    <attribute name="history" sql="select history from $instance"/>  --????
+    <attribute name="geospatial_lat_min">
+      <sql>select min("LATITUDE") from $instance</sql>
+    </attribute>
+    <attribute name="geospatial_lat_max">
+      <sql>select max("LATITUDE") from $instance</sql>
+    </attribute>
+    <attribute name="geospatial_lon_min">
+      <sql>select min("LONGITUDE") from $instance</sql>
+    </attribute>
+    <attribute name="geospatial_lon_max">
+      <sql>select max("LONGITUDE") from $instance</sql>
+    </attribute>
+    <attribute name="geospatial_vertical_min" value="0"/>
+    <attribute name="geospatial_vertical_max" value="0"/>
+    <attribute name="time_coverage_start">
+      <sql>select to_char(min("TIME" AT TIME ZONE 'UTC'), 'YYYY-MM-DD"T"HH24:MI:SS"Z"') from $data</sql>
+    </attribute>
+    <attribute name="time_coverage_end">
+      <sql>select to_char(max("TIME" AT TIME ZONE 'UTC'), 'YYYY-MM-DD"T"HH24:MI:SS"Z"') from $data</sql>
+    </attribute>
+    <attribute name="data_centre" value="'eMarine Information Infrastructure (eMII)'"/>
+    <attribute name="data_centre_email" value="'info@emii.org.au'"/>
+    <attribute name="institution_references" value="'http://www.imos.org.au/emii.html'"/>
 
+    <attribute name="citation">
+      <value>'The citation in a list of references is: "IMOS [year-of-data-download], [Title], [data-access-url], accessed [date-of-access]"'</value>
+    </attribute>
+    <attribute name="acknowledgement">
+      <value>'Any users of IMOS data are required to clearly acknowledge the source of the material in the format: "Data was sourced from the Integrated Marine Observing System (IMOS) - an initiative of the Australian Government being conducted as part of the National Collaborative Research Infrastructure Strategy and and the Super Science Initiative."'</value>
+    </attribute>
+    <attribute name="distribution_statement">
+      <value>'Data may be re-used, provided that related metadata explaining the data has been reviewed by the user, and the data is appropriately acknowledged. Data, products and services from IMOS are provided "as is" without any warranty as to fitness for a particular purpose.'</value>
+    </attribute>
+    <attribute name="project_acknowledgement">
+      <value>'The collection of this data was funded by IMOS'</value>
+    </attribute>
+  </globalattributes>
+  <variables>
     <variable>
-      <name>id</name>
+      <name>trajectory</name>
```

Comment:
The netcdf generator is only going to output one file per dataset so we don't need this extra variable. I didn't use the equivalent one for anmn_ts.